Bowling/Frame.cs

```diff
@@ -0,0 +1,77 @@
+﻿namespace Game
+{
+    public class Frame
+    {
+        private int firstShot;
+        private int secondShot;
+        private bool firstShotUsed = false;
+        private bool secondShotUsed = false;
+
+        public int FirstShot
+        {
+            get
+            {
+                return firstShot;
+            }
+            set
+            {
+                firstShot = value;
+            }
+        }
+        public int SecondShot
+        {
+            get
+            {
+                return secondShot;
+            }
+            set
+            {
+                secondShot = value;
+            }
+        }
+
+        public Frame(int firstShotScore = 0, int secondShotScore = 0)
```

Comment:
If someone will try to use the constructor with negative values the game will still work?